lib/src/base-styles/_nx-variables.scss

```diff
@@ -65,13 +65,21 @@
                             0 1px 2px 0 rgba(91, 95, 141, 0.2),
                             0 1px 1px 0 rgba(124, 128, 179, 0.2);
 
+  // Radios and Checkboxes
+  --nx-radio-checkbox-border-width: 1px;
+  --nx-radio-checkbox-border-width-disabled: 1px;
```

Comment:
Please follow the naming conventions already present in this file e.g. `--nx-border-width-radio-checkbox`

Reply:
> Please follow the naming conventions already present in this file e.g. `--nx-border-width-radio-checkbox`

Renamed here: fa351f5ea157e03267d26df3eebf91cfb2881237